### DISCUSSION TITLE
Display Datetime in Calendar Template

### DISCUSSION THREAD

**Discussion by Alessandro-Avallone-Developer** [directus/directus#12886] [Archive]
Hi,

in calendar template i would like to have hours in format HH:MM even if datetime has minute 00:00
![image](https://user-images.githubusercontent.com/52598218/164280228-619e86df-8ec4-4c59-bb84-e0cf3dde02ac.png)

**Comment by azrikahar** [comment]
Seems to be configurable via `views` as described here: https://fullcalendar.io/docs/view-specific-options

End result of omitting the [default `omitZeroMinute` flag](https://github.com/fullcalendar/fullcalendar/blob/f65d1fb3ee6e8b47ae8ff31a0b6292c35d183080/packages/daygrid/src/event-rendering.ts#L7):

![chrome_bvbNPSRsCN](https://user-images.githubusercontent.com/42867097/164463752-25d24422-31d1-4e7b-9fca-e3e58f0ee0b9.png)

Do we want to make this configurable? Might be difficult 🤔

**Reply by rijkvanzanten** [reply]
I think in this case, we can default it to enabled. I think it's better to always show the full time for consistencies sake. We could make it configurable, but it feels a little overkill to have that as an option in the sidebar 🤔

**Reply by azrikahar** [reply]
Yeah agreed. Just a slight concern of user that _wants_ it to not show the full time (if any?), but this configuration topic probably goes back to #5842 as a whole, so I'll make the PR to enable this by default for now 👍

**Reply by Alessandro-Avallone-Developer** [reply]
OK thanks for replies ... so i suppose there is no way for the moment to change this behaviour
i'm on directus cloud so can't make any changes to source code

**Reply by rijkvanzanten** [reply]
> OK thanks for replies ... so i suppose there is no way for the moment to change this behaviour
> 
> i'm on directus cloud so can't make any changes to source code

We just changed the defaults to your suggestion, so this will launch to the platform shortly 😄

**Comment by Alessandro-Avallone-Developer** [comment]
Great !!! 😍

Il giorno gio 21 apr 2022 alle ore 18:18 Rijk van Zanten <
***@***.***> ha scritto:
